x-pack/plugins/cases/server/telemetry/queries/utils.ts

```diff
@@ -125,14 +134,20 @@ export const getCountsAndMaxData = async ({
   savedObjectsClient,
   savedObjectType,
   filter,
+  additionalAggs = {},
 }: {
   savedObjectsClient: TelemetrySavedObjectsClient;
   savedObjectType: string;
   filter?: KueryNode;
+  additionalAggs?: Record<string, estypes.AggregationsAggregationContainer>;
 }) => {
   const res = await savedObjectsClient.find<
     unknown,
-    { counts: Buckets; references: MaxBucketOnCaseAggregation['references'] }
+    {
+      counts: Buckets;
+      references: MaxBucketOnCaseAggregation['references'];
+      uniqueAlertCommentsCount: { value: number };
```

Comment:
I think we need some different logic to name these results. `uniqueAlertCommentsCount` is specific to the aggregation you passed so if someone else uses `getCountsAndMaxData` and passes a different `additionalAggs` I think this might not work.

Reply:
Changed it, thanks!